src/hardware/introduction.md

```diff
@@ -1,15 +1,28 @@
 
-## Input-Output Expander (IOX)
+When using Geotab's Hardware Add-On protocols, it is important not to send excessive amounts of data to the GO device.  This can be assured by only sending data when it is completely necessary and avoiding the sending of repetitive values. The GO device will stop recording data from the Hardware Add-On device if excessive Add-On data is detected. When this condition occurs, "Status data no longer being saved due to excessive data" will appear in the Fault Data for the GO device within MyGeotab. In order to re-enable data logging for the Hardware Add-On, the GO device will need to be completely disconnected from power and then powered back up again.
 
-Third-party devices communicate with the GO device using an IOX Add-On corresponding to the aforementioned interfaces, namely the [IOX-CAN](https://docs.google.com/document/d/19Wlwsb_AnpE3AndOf1cbiFLzTOCI5fwscOwQh5a6jF8/preview), [IOX-USB](https://docs.google.com/document/d/1_vocie4MYAHAUBOJ_AUXzYN11jrR5jyCynFgbqigFys/preview), [IOX-RS232](https://docs.google.com/document/d/1UDEwQOY2zH1ABQ3UP6rdlcZ--LPt5SUbKEZ_AZgUsWk/preview) and [IOX-BT](https://docs.google.com/document/d/1ICzpfhyYQEl1acQtJ9AGw09pzH5ADHNC8EGIcvTP2KU/preview).
+## Integrating with the Input-Output Expander (IOX)
 
-Third-party data sent to the GO device will be transmitted to the MyGeotab platform. Data received from the third-party device can be queried within MyGeotab by navigating to **Engine &amp; Maintenance** > **Engine &amp; Device…** > **Measurements**. It can also be reported on just like any other status data.
\ No newline at end of file
+Hardware Add-Ons communicate with the GO device using an IOX corresponding to the aforementioned interfaces, namely the [IOX-CAN](https://docs.google.com/document/d/19Wlwsb_AnpE3AndOf1cbiFLzTOCI5fwscOwQh5a6jF8/preview), [IOX-USB](https://docs.google.com/document/d/1_vocie4MYAHAUBOJ_AUXzYN11jrR5jyCynFgbqigFys/preview), [IOX-RS232](https://docs.google.com/document/d/1UDEwQOY2zH1ABQ3UP6rdlcZ--LPt5SUbKEZ_AZgUsWk/preview), [IOX-BT](https://docs.google.com/document/d/1ICzpfhyYQEl1acQtJ9AGw09pzH5ADHNC8EGIcvTP2KU/preview) and any [Custom IOX](../developing-an-iox/hardware-design-guide/). The format of data sent through an IOX could vary with the type of IOX, which is shown in the table below. All IOXs support data transfer from Add-On to GO device, while some IOXs also support data transfer from GO device to Add-On.
+
+| IOX Type | Add-On Protocol | Status Data | Custom Data | Binary Data | Data Transfer From GO Device To Add-On   |
+| :---: | :---: | :---: | :---: | :---: | :---: |
+| IOX-RS232 | Y | Y | Y | Y | Y |
+| IOX-USB | Y | Y | Y | Y | Y |
+| IOX-CAN | Y | Y | N | N | Y |
+| IOX-BT | Y | Y | Y | N | N |
+| IOX-ANAGLOG | N | Y | N | N | N |
```

Comment:
Should Analog be in this list?

Reply:
What makes IOX-ANALOG different such that you don't think it belongs to the list?

Reply:
Every IOX logs status data. The difference IOX-ANALOG, and all those other IOXs, are not covered by any of our documentation here.

Reply:
Misspelled analog as "ANAGLOG".

Reply:
Good eye Justin, thanks. I will correct in a later commit.
I think we should keep IOX-ANALOG in the table: this gives integrator visibility on the differences between IOX-ANALOG and anything else. Such information is not available anywhere. We should add a page to explain IOX-ANALOG in the future.

Reply:
@jwidz Based on your comment that "IOX-ANALOG isn't in the scope of HDK integration", are you recommending to remove it from the table too?

Reply:
IOX-ANALOG will be removed in a future PR

Reply:
Don't want to remove now?

Reply:
The latest commit has removed IOX-ANALOG row